common/src/test/scala/org/apache/celeborn/common/meta/WorkerInfoSuite.scala

```diff
@@ -303,4 +303,34 @@ class WorkerInfoSuite extends CelebornFunSuite {
       }
     }
   }
+
+  test("Test WorkerInfo hashcode") {
+    val host = "127.0.0.1"
+    val rpcPort = 9082
+    val pushPort = 9092
+    val fetchPort = 9093
+    val replicatePort = 9094
+    val workerInfo = new WorkerInfo(host, rpcPort, pushPort, fetchPort, replicatePort)
+
+    // origin hashCode()
+    var now = System.nanoTime()
+    val state = Seq(host, rpcPort, pushPort, fetchPort, replicatePort)
+    val originHash = state.map(_.hashCode()).foldLeft(0)((a, b) => 31 * a + b)
+    var after = System.nanoTime()
+    println("WorkerInfo origin hash result = " + originHash + ", costs " + (after - now))
```

Comment:
avoid call `println` in UTs

Reply:
use `log` or `assert(condition: Boolean, clue: Any)` instead of `println`.

Reply:
Thanks. I find no need to print logs. Remove done.